mdoc/Mono.Documentation/MDocUpdater.cs

```diff
@@ -327,6 +355,12 @@ public override void Run (IEnumerable<string> args)
             if (!string.IsNullOrWhiteSpace (FrameworksPath))
                 frameworks.WriteToDisk (srcPath);
 
+            if (statisticsEnabled)
+            {
+                File.WriteAllText (statisticsFilePath, StatisticsFormatter.Format(statisticsCollector.Storages));
```

Comment:
What happens if `statisticsFilePath` doesn't exist? If it automatically creates the file, what happens if the filepath leads to a set of folders that don't exist? What happens if `statisticsFilePath` just points to a folder ... perhaps you could append a default filename?